src/node_file.cc

```diff
@@ -460,28 +460,28 @@ void Close(const FunctionCallbackInfo<Value>& args) {
 }  // anonymous namespace
 
 void FillStatsArray(double* fields, const uv_stat_t* s) {
-  fields[0] = s->st_dev;
-  fields[1] = s->st_mode;
-  fields[2] = s->st_nlink;
-  fields[3] = s->st_uid;
-  fields[4] = s->st_gid;
-  fields[5] = s->st_rdev;
+  fields[0] = static_cast<double>(s->st_dev);
```

Comment:
These changes seem unrelated, and I don’t think they are actually changing anything?